deploy/crds/kieapp.crd.yaml

```diff
@@ -182,6 +182,9 @@ spec:
                           type: object
                           description: GitHooks configuration object
                           properties:
+                            sshSecret:
+                              type: string
+                              description: Secret to use for ssh key(s) mount.
```

Comment:
The (s) is incorrect as we only use one key. Suggestion:
Secret to use for ssh key and known hosts file